UnitySDK/ProjectSettings/ProjectSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!129 &1
 PlayerSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 14
+  serializedVersion: 15
```

Comment:
Undo this? And ProjectVersion.txt and *.asset?

Reply:
👍